test-tests/shared/src/main/scala/zio/test/TestMain.scala

```diff
@@ -1,10 +1,10 @@
 package zio.test
 
-import scala.concurrent.ExecutionContext.Implicits.global
-
-import zio.test.mock.MockSpecSpec
+import zio.test.TestUtils.{report, scope}
```

Comment:
This needs a `fmt`.

Reply:
I had "optimize imports" enabled for committing... unfortunately it "optimizes" the formatting too :/ I've disabled this now